README.md

```diff
@@ -144,3 +144,22 @@ Please contact us for specific advice, but in outline this generally involves:
 ## Monitoring and logging
 
 Please see the [monitoring-and-logging.README.md](docs/monitoring-and-logging.README.md) for details.
+
+## CI/CD automation
+
+A GitHub Actions workflow which checks for new upstream version release tags and creates a PR to update the downstream repo, can be found at:
+
+        .github/workflows/upgrade-check.yml.sample
+
+If activated, the workflow is scheduled by default to run every day at 9 AM UTC and can be triggered manually via the `workflow_dispatch` event. How to activate the workflow is detailed at the top of the file.
+
+Workflow uses [create-pull-request](https://github.com/peter-evans/create-pull-request) to handle the pull request action. See for action inputs.
+
+In order for GitHub actions to fetch workflow changes in `.github/workflows`, a PAT for each deployment must be provided.
```

Comment:
I don't think this is true. The PAT isn't required to *fetch* workflow changes, it's to create PRs which may involve modifying workflows (as far as I understand it anyway). Otherwise, the standard token provided to the workflow by GitHub actions would be sufficient.

We should also rephrase the discussion of tokens to state that both fine-grained tokens and user PAT tokens should work but that it was tested with a PAT token. Fine-grained repo-scoped token being preferred if possible.